tensorflow/lite/micro/tools/ci_build/test_xtensa_hifi3z.sh

```diff
@@ -50,6 +50,20 @@ if [[ ${1} == "INTERNAL" ]]; then
     OPTIMIZED_KERNEL_DIR=xtensa \
     XTENSA_CORE=HIFI_190304_swupgrade \
     test_integration_tests_seanet_conv -j$(nproc)
+
+  readable_run make -f tensorflow/lite/micro/tools/make/Makefile \
```

Comment:
These tests will fail on GH right? Should they be protected behind a flag of some sort, or perhaps only be used in the internal presubmit?

Reply:
Yes, they are protected behind "if [[ ${1} == "INTERNAL" ]]; then" and we only append "INTERNAL" in internal tests.